backend/django/core/templates/projects/detail.html

```diff
@@ -48,7 +48,9 @@ <h5 class="panel-title">
             <div class="panel-body">
               <p>Fully Labeled: <span id="fully-labeled"></span></p>
               <p>Unlabeled and Unassigned: <span id="unlabeled-unassigned"></span></p>
-              <p>Other (Partially Labeled/IRR/Skipped): <span id="other-labeled"></span></p>
+              <p>Awaiting Adjudication: <span id="awaiting-adjudication"></span></p>
+              <p>Recycle Bin: <span id="recycled"></span></p>
+              <p>Partially Labeled/IRR): <span id="other-labeled"></span></p>
```

Comment:
Two notes about the other-labeled group:

1. There's a ) without a partner
2. This group also includes assigned data which has been sent to someone but not labeled yet